lib/saleorApp.ts

```diff
@@ -0,0 +1,5 @@
+import { FileAPL, VercelAPL } from "@saleor/app-sdk/APL";
+
+// For local development store auth data in the `.auth-data.json`.
+// For app deployment on Vercel with Saleor CLI, use vercelAPL.
```

Comment:
This is important part, lest
1. use block comment to improve IDE support
2. Add link to docs what is APL

In the future I'd like to introduce some entry point (root file) that will have a single app constructur/creator, but now its fine to be here